README.md

```diff
@@ -190,6 +190,14 @@ appInsights.downloadAndSetup({
 ```
 Test in internal environment to verify monitoring telemetry is working as expected. If all works, please update your api signatures appropriately to SDK V2 version and deploy in your production environments.
 
+## Available extensions for the SDK
+
+
+| Extensions |
+|------|---------|-------------|
+| [React](https://github.com/Microsoft/ApplicationInsights-JS/tree/master/vNext/extensions/applicationinsights-react-js) 
```

Comment:
this table is not rendered properly
![image](https://user-images.githubusercontent.com/1350095/56949323-81ae6800-6b2a-11e9-9b26-7c2bd3db5a96.png)

Reply:
Also, I suggest to move the extension section up in the readme, have a table of contents and/or use `<details><summary></summary></details>` to make the readme more readable. See https://gist.github.com/ericclemmons/b146fe5da72ca1f706b2ef72a20ac39d